core/src/main/java/com/datastax/oss/driver/api/core/cql/StatementBuilder.java

```diff
@@ -157,7 +157,13 @@ public SelfT setIdempotence(@Nullable Boolean idempotent) {
   /** @see Statement#setTracing(boolean) */
   @NonNull
   public SelfT setTracing() {
-    this.tracing = true;
+    return setTracing(true);
```

Comment:
Maybe we should add a short explanation in the javadoc of this method, along the lines of: "this method exists for historical reasons, it is just a shortcut for setTracing(true)".

Reply:
Agreed, this seems reasonable... will change.